ops/mappings.json

```diff
@@ -133,6 +133,13 @@
           "dao": "package",
           "mapping": "DAOFactory",
           "arcVersion": "0.1.2-rc.1"
+        },
+        {          
+          "name": "TokenTrade",
+          "contractName": "TokenTrade",
+          "dao": "package",
+          "mapping": "TokenTrade",
+          "arcVersion": "0.1.2-rc.2"
```

Comment:
This version might change

Reply:
Yes. Also please make sure to add this to Ganache, Rinkeby, Kovan, xDai.